src/utils/jwt.go

```diff
@@ -0,0 +1,90 @@
+package utils
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"golang.org/x/oauth2"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+// TODO: This code was taken from https://gist.github.com/kadel/c30b3085e2e90a93393b99a2b39f4806. if possible, find a leaner way to obtain the token.
```

Comment:
Please let's not consider it as TODO if it works, let's keep it. Also, I'd add this comment before `package utils` above as we consider that the entire `jwt.go` was copied (including dependencies).

Reply:
I kept it in the godoc of the function since I have made adjustments to this code.

Reply:
Thanks! :)